finetune/adapter.py

```diff
@@ -98,20 +97,21 @@ def main(fabric: L.Fabric, data_dir: Path, checkpoint_dir: Path, out_dir: Path):
     config = Config.from_name(name=checkpoint_dir.name)
     checkpoint_path = checkpoint_dir / "lit_model.pth"
     fabric.print(f"Loading model {str(checkpoint_path)!r} with {config.__dict__}")
-    with fabric.init_module(empty_init=False):
+    with fabric.init_module(empty_init=True):
```

Comment:
Why can `empty_init=True` be used here now?

Reply:
The idea is that `empty_init=True` will instantiate with `torch.device("meta")` and `fabric.setup_module()` will materialize any leftover parameters, but this isn't implemented yet.